pmm/pmm-submodules.groovy

```diff
@@ -0,0 +1,142 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+pipeline {
+    agent {
+        label 'cli'
+    }
+    parameters {
+        string(
+            defaultValue: 'PMM-2.0',
+            description: 'Tag/Branch for pmm-submodules repository',
+            name: 'GIT_BRANCH')
+        string(
+            defaultValue: '',
+            description: 'URL for pmm-submodules repository PR',
+            name: 'CH_URL')
+        string(
+            defaultValue: '',
+            description: 'ID for pmm-submodules repository PR',
+            name: 'CH_ID')
+        choice(
+//            default is choices.get(0) - el9
+            choices: ['el9', 'el7'],
+            description: 'Select the OS to build for',
+            name: 'BUILD_OS')
+    }
+    environment {
+        PATH_TO_SCRIPTS = 'sources/pmm/src/github.com/percona/pmm/build/scripts'
+        PMM_VERSION = """${sh(returnStdout: true, script: "cat VERSION").trim()}"""
```

Comment:
```suggestion
        PMM_VERSION = sh(returnStdout: true, script: "cat VERSION").trim()
```

Reply:
Would the this work? I'm trying to suggest the same syntax as everywhere else, just to be consistent.